doc/backing-store-crd.md

```diff
@@ -31,23 +41,22 @@ metadata:
   finalizers:
   - noobaa.io/finalizer
   labels:
-    app: noobaa
-  name: bs
-  namespace: noobaa
+    app: <>
```

Comment:
How does the user change the name of the `app` using the CLI?

Reply:
So it should stay `noobaa` since in line 28 it was defined that `<>` depends on user choice.

> From here on out, fields that depend on the user's choice and input will be marked with `<>`.

Reply:
I marked the `app: <>`, as you said the users cannot change it so it is not the case of `<>` if I understand you correctly.